src/clr-angular/data/datagrid/datagrid-action-overflow.ts

```diff
@@ -50,6 +76,21 @@ export class ClrDatagridActionOverflow implements OnDestroy {
     if (boolOpen !== this._open) {
       this._open = boolOpen;
       this.openChanged.emit(boolOpen);
+      if (boolOpen && isPlatformBrowser(this.platformId)) {
+        this.zone.runOutsideAngular(() => {
+          // It emits 2+ onStable events in a synchronous sequence. We need to wait for the last one.
+          // Some of them are probably only in dev mode, but some are caused by collapsing the previous overflow window.
+          this.zone.onStable
+            .asObservable()
+            .pipe(debounceTime(0), take(1))
+            .subscribe(() => {
+              const firstButton = this.ref.nativeElement.querySelector(`.${this.overflowClassName} button`);
```

Comment:
I think we should be a bit stricter to select the buttons here ( `button.action-item` ) since we recommend buttons with `action-item` class?

Reply:
In a previous PR, on another issue, I was required to remove the specific implementation (button, div, whatever) from a selector and stick to using the class-name, as behavior identifying but not bound to implementation. I started to agree with this.
Why you don't think that a class name is enough?

Reply:
Because users can project any elements they want there. In our document, we recommend projecting buttons with a class .action-item, but we don't forbid them from using their completely customized actionable items. If they have their own checkboxes and buttons, our current implementation could set the focus on the wrong element. So in order to prevent conflicts with app's implementation there, we should set this default focus behavior only on what we recommend.

Reply:
If they put buttons without the .action-item class, our focus will be the last problem to worry about:
<img width="329" alt="Screen Shot 2019-06-26 at 2 26 34 PM" src="https://user-images.githubusercontent.com/1798828/60176297-95223a80-981e-11e9-9b2a-c37e65d8b3c1.png">

I added the class to the selector, though I don't think that this will be the exact reason for someone to decide to follow our recommendations.